### ISSUE TITLE
Crash when xin/xout not given for UDO

### ISSUE THREAD

**Issue by kunstmusik** [csound/csound#427]
The following UDO crashes Csound due to an xout not being given. 

```
<CsoundSynthesizer>
<CsOptions>
-+rtaudio=jack -odac -b4096
</CsOptions>
<CsInstruments>
sr     =     44100
ksmps     =     32
nchnls     =     2
0dbfs    =    1

 opcode SimpleTest, k, a
     setksmps 1
     aInput xin
 endop


instr 1
a1 init 0
k1 SimpleTest a1
endin

</CsInstruments>
<CsScore>
i 1 0 1
</CsScore>
</CsoundSynthesizer>
```

**Comment by kunstmusik** [comment]
Note, should be fixed as part of #310

**Comment by kunstmusik** [comment]
This will be released as part of 6.05.0.
